src/main/markdown/doc/latest/DevGuideI18nLocale.md

```diff
@@ -76,9 +76,13 @@ are probably better off choosing some real language as a default, such as
 the following in your [module XML](DevGuideOrganizingProjects.html#DevGuideModuleXml):
 
 ```
-<set-property-fallback name="locale" value="en"/>
+<extend-property name=locale" value"es,de,fr,en,it" <!-- 1. full list of supported locales --> 
```

Comment:
Missing `"`  in `name` attribute value. Missing `=` (also `values` vs `value`). Missing `/>`.